python/orca/src/bigdl/orca/learn/pytorch/pytorch_pyspark_estimator.py

```diff
@@ -186,7 +186,8 @@ def fit(self,
             feature_cols=None,
             label_cols=None,
             validation_data=None,
-            callbacks=[]):
+            callbacks=[],
+            recollate_fn=empty_recollate_fn):
```

Comment:
shall simply call it `collate_fn`? @hkvision

Reply:
Yes, I think `collate_fn` is better.